test/generators/Generators.scala

```diff
@@ -58,16 +49,21 @@ trait Generators extends ModelGenerators with JsonGenerators {
       choose(0, vector.size - 1).flatMap(vector(_))
     }
 
-  private val validEmailGen: Gen[String] = RegexpGen.from("[a-zA-Z0-9\\.!#$%&'*+/=?^_`{|}~-]{1,25}@[a-zA-Z0-9-.]{1,25}")
+  def eoriString: Gen[String] = alphaNumString(12)
```

Comment:
This is actually incorrect. EORI has specific format so it should not be just alphanumeric.
According to the validation in Declarations service it is: 2 letters followed by 10 to 15 digits.

Reply:
:+1: